### ISSUE TITLE
latex: fail to generate .tex in some languages

### ISSUE THREAD

**Issue by shimizukawa** [sphinx-doc/sphinx#432]
I tested make latex with language=ja, zh_CN, tr. But it will fail to generate .tex file. (error related with Babel is only occurred in 'ja')

en, fr, de are OK.

And more, I think it was "Error" because the task was stopped. But following message is "warning". 

```
#!bash

writing... WARNING: no Babel option known for language 'ja'
WARNING: a Unicode error occurred when writing the output. Please make sure all config values that contain non-ASCII content are Unicode strings.
copying images... themes/default.png themes/haiku.png themes/sphinxdoc.png themes/traditional.png themes/scrolls.png more.png themes/nature.png themes/agogo.png
copying TeX support files... done
build succeeded, 2 warnings.

Build finished; the LaTeX files are in _build/latex.
Run `make all-pdf' or `make all-ps' in that directory to run these through (pdf)latex.
```

---
- Bitbucket: https://bitbucket.org/birkenfeld/sphinx/issue/432
- Originally reported by: SHIBUKAWA Yoshiki
- Originally created at: 2010-05-27T23:57:32.974

**Comment by shimizukawa** [comment]
_From Georg Brandl on 2010-05-28 20:39:49+00:00_

Thanks for the report; both issues should be fixed in <<cset c0a57a84ca1e066831c9ed55134179b83ebabad8>>.

**Comment by shimizukawa** [comment]
_From Georg Brandl on 2011-09-22 10:14:45+00:00_

Removing milestone: 1.0 (automated comment)
